src/input/ansi.rs

```diff
@@ -0,0 +1,23 @@
+//! This module provides input related ANSI escape codes.
+
+use crate::csi;
+
+pub(crate) fn enable_mouse_mode_csi_sequence() -> String {
```

Comment:
This might as well be a constant right? Then we can remove the string alocation completely. I am not sure if we can make `AnsiType` in command be a `&' static str` then we can get rid of this allocation completely.